.travis.yml

```diff
@@ -8,12 +8,10 @@ node_js:
 env:
   matrix:
   - TEST_BROWSER_NAME=PhantomJS
-  - TEST_BROWSER_NAME=chrome TEST_BROWSER_OS='Windows 7,Linux'
-  - TEST_BROWSER_NAME=firefox TEST_BROWSER_OS='Windows 7,Linux' TEST_BROWSER_VERSION='30,26'
-  - TEST_BROWSER_NAME='internet explorer' TEST_BROWSER_OS='Windows 7' TEST_BROWSER_VERSION=10
-  - TEST_BROWSER_NAME='internet explorer' TEST_BROWSER_OS='Windows 8.1' TEST_BROWSER_VERSION=11
-  - TEST_BROWSER_NAME=safari TEST_BROWSER_OS='OS X 10.8' TEST_BROWSER_VERSION=6
-  - TEST_BROWSER_NAME=safari TEST_BROWSER_OS='OS X 10.9' TEST_BROWSER_VERSION=7
+  - TEST_BROWSER_NAME=chrome TEST_BROWSER_OS='Windows 10,Linux,OS X 10.11'
+  - TEST_BROWSER_NAME=firefox TEST_BROWSER_OS='Windows 10,Linux,OS X 10.11'
+  - TEST_BROWSER_NAME='internet explorer' TEST_BROWSER_OS='Windows 10'
+  - TEST_BROWSER_NAME=safari TEST_BROWSER_OS='OS X 10.11'
```

Comment:
It's more of a sauce labs things. They only seem to be updating the latest versions properly.